src/components/archive.rs

```diff
@@ -118,3 +118,127 @@ where
         Ok(())
     }
 }
+
+/// An archive for storing individuals between operators, e.g. for subsequent calculation of measures.
+#[derive(Default, Tid)]
+pub struct IntermediateArchive<P: Problem + 'static>(Vec<Individual<P>>);
```

Comment:
It would maybe be nicer to add a `Archive<I: Identifier>` struct that is used by all archive components that require a `Vec<Individual>`, but I think this is fine.